gymnasium_robotics/envs/maze/maze.py

```diff
@@ -262,7 +262,9 @@ def generate_reset_pos(self) -> np.ndarray:
 
         # While reset position is close to goal position
         reset_pos = self.goal.copy()
-        while np.linalg.norm(reset_pos - self.goal) <= 0.5:
+        while (
+            np.linalg.norm(reset_pos - self.goal) <= 0.5 * self.maze.maze_size_scaling
```

Comment:
Shouldn't it be `self.maze._maze_size_scaling`?

Reply:
This is okey. The attribute is wrapped with `@property` https://github.com/Farama-Foundation/Gymnasium-Robotics/blob/5a13eaae5e56b48c0c61833b7365fe114c249dcb/gymnasium_robotics/envs/maze/maze.py#L71